site/_resources/envoy.md

```diff
@@ -53,3 +53,14 @@ If you are providing your own Envoy it must be compiled with the following exten
 [5]: https://groups.google.com/forum/#!msg/envoy-announce/sVqmxy0un2s/8aq430xiHAAJ
 [6]: https://groups.google.com/d/msg/envoy-announce/y4C7hXH6WrU/eRoMZ6WaAgAJ
 [7]: https://groups.google.com/d/msg/envoy-announce/qrrF8klFl-I/nz12XtqmAAAJ
+
+## Envoy TCP keepalives
+
+Contour configures TCP keepalives for Envoy listener sockets so that half-open connections with clients (downstream in Envoy speak) are flushed which helps Envoy work better with networks that drop long running idle TCP connections.  Once a TCP connection has been idle for 45 seconds a keepalive packet is sent every 5 seconds and if 9 packets receive no response then the connection will be terminated by Envoy.
+
+TCP keepalives are also enabled for connections from Envoy to servers (upstream in Envoy terminology). Once a connection has been idle for 30 seconds a keepalive packet is sent every 5 seconds and if 3 packets receive no response then the connection will be terminated by Envoy.
+
+Envoy is only supported on Linux so Contour always configures Linux TCP keep-alive socket options regardless of the platform that Contour is running on.
+
+Note that TCP keepalives are handled entirely at the network layer and require no application configuration.  They are not the same as HTTP Keep-Alive or application level heartbeats.
+
```

Comment:
Unfortunately, this doc is the Envoy support matrix, so it's not the right place to document operational or configuration concerns. Can you please remove this for now?

Reply:
Ok, done.